packages/documentation/src/stories/utilities/vertical-align/vertical-align.snapshot.stories.ts

```diff
@@ -0,0 +1,34 @@
+import type { StoryObj, StoryFn, StoryContext } from '@storybook/web-components';
+import { html } from 'lit';
+import meta from './vertical-align.stories';
+import './vertical-align.styles.scss';
+
+const { id, ...metaWithoutId } = meta;
+
+console.log(meta?.argTypes?.align?.options);
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const VerticalAlign: Story = {
+  render: () => {
+    return html`
+      ${meta?.argTypes?.align?.options?.map(
+        align =>
+          html`<div>
+            <img class="logo" alt="logo" src="/assets/images/logo-swisspost.svg" />
+            <span class="align-${align}">${align ? `align-${align}` : 'text'}</span>
```

Comment:
We want to show how the image can be aligned with the text, not the other way around.

```suggestion
            <span>${align ? `align-${align}` : 'text'}</span>
            <img class="logo align-${align}" alt="logo" src="/assets/images/logo-swisspost.svg" />
```